index.js

```diff
@@ -10,7 +10,7 @@ try {
   const product = input1 * input2;
 
   // Create a custom message
-  const message = `The inputs are ${input1} and ${input2}. Sum is ${sum}, product is ${product}.`;
```

Comment:
The variable `foo` is likely undefined or misspelled. Double check that you meant to use this variable name in the `message` string.